src/partials/meet_our_teaches.html

```diff
@@ -0,0 +1,83 @@
+<section class="teachers-section" id="teachers">
+  <div class="container">
+    <h2 class="section-title">
+      Meet our <span class="section-title-accent">teachers</span>
+    </h2>
+    <p class="section-text">
+      Discover our team of dedicated educators, committed to guiding you on your
+      journey to English proficiency with personalized and engaging instruction
+    </p>
+    <ul class="teachers-list">
+      <li class="teacher-card">
+        <img
```

Comment:
It would be better to use wrapper around a img tag e.g. 
<div class="wrapper">
    <img ... />
</div>